src/eqProps.js

```diff
@@ -14,7 +14,7 @@ var equals = require('./equals');
  * @param {Object} obj1
  * @param {Object} obj2
  * @return {Boolean}
- *
+ * @deprecated since v0.18.0
```

Comment:
I agree that `R.always(true)` is clearer than `R.T`. I wouldn't object to removing `R.T` and `R.F`.

`R.propEq` and `R.propIs` are both unnecessary. Both functions are useful, though. It depends on the domain whether one is more likely to ask _are you 42?_ (a value-level enquiry) or _are you a Number?_ (a type-level enquiry).